test/ui/suite/kubernetesContext.ts

```diff
@@ -22,17 +26,35 @@ export function kubernetesContextTest() {
         let quickPicks: QuickPickItem[];
         const allQuickPicksTexts: string[] = [];
 
+        const kubeCopy = `${getKubeConfigPath()}-cp`
+
         before(async function() {
             view = await (await new ActivityBar().getViewControl(VIEWS.openshift)).openView();
             explorer = await view.getContent().getSection(VIEWS.appExplorer);
+
+            //make kubeconfig copy
+            fs.copySync(getKubeConfigPath(), kubeCopy);
+
             await activateCommand('>OpenShift: Log out');
+
```

Comment:
@lgrossma Could you please trim unneeded extra empty lines before we merge it?

Reply:
Formatted the documents with vscode built-in feature. Hope it is better now.